### PR TITLE
Reports agent version as a label on every span

### LINKED ISSUE #110: Report agent information for tracking metrics
Add the label "trace.cloud.google.com/agent" with format "\<name\> \<version\>" for tracking.

### PR DESCRIPTION
**Pull request by matthewloring**
Fixes #110.

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots PTAL.

File: lib/trace-writer.js [hunk 1]
```diff
@@ -74,6 +74,7 @@ TraceWriter.prototype.writeSpan = function(spanData) {
   if (this.config_.instanceId) {
     spanData.addLabel(traceLabels.GCE_INSTANCE_ID, this.config_.instanceId);
   }
+  spanData.addLabel(traceLabels.AGENT_DATA, 'node ' + pjson.version);
```
Comment (ofrobots) [inline lib/trace-writer.js]: Can we use the package name instead (or in addition to) of just `node`?

**Comment by ofrobots** [comment]
1. See the duplication here: https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/blob/master/lib/trace-writer.js#L28. Please reconcile.

2. It makes sense to put this (and while we are at it, the hostname and instance id) in the default headers since these don't vary for the lifetime of the app.

File: lib/trace-writer.js (hunk 1 above)
Comment (matthewloring) [inline lib/trace-writer.js]: The name currently is "@google/cloud-trace" which doesn't carry any language specific information (and I don't think it necessarily should). The fact that this is a trace agent can be inferred from the fact that we're reading the information from a trace span.

**Comment by matthewloring** [comment]
1. I am not sure what format the consumer of the header is expecting. I can modify the new label we attach, my hope was to keep the string as small as possible since this label will be appearing on every single trace but I can use the longer form included in the header if preferable.

2. What is the benefit of adding information to the headers without any intended consumers of that information?

**Comment by matthewloring** [comment]
@ofrobots Default headers have been removed as discussed yesterday. Ready for another look.

File: lib/trace-writer.js (hunk 1 above)
Comment (ofrobots) [inline lib/trace-writer.js]: We agreed that keeping the name shorter is better for now.

**Comment by ofrobots** [comment]
LGTM.

**Pull request merged**